### ISSUE TITLE
Authors are interpreted as subgenera

### ISSUE THREAD

**Issue by KatjaSchulz** [gnames/gnparser#265]
These are all valid/accepted names from the current version of the Catalogue of Life

Plant genera
[Nassella (Trin.) É.Desv.](https://www.catalogueoflife.org/data/taxon/8VZLC) – simple: Trin. – full: Nassella subgen. Trin.
[Dacrycarpus (Endl.) de Laub.](https://www.catalogueoflife.org/data/taxon/C6Y3P) – simple: Endl. – full: Dacrycarpus subgen. Endl.
[Lysiphyllum (Benth.) de Wit](https://www.catalogueoflife.org/data/taxon/9CL8W) – simple: Benth. – full: Lysiphyllum subgen. Benth.
[Tricholemma (Röser) Röser](https://www.catalogueoflife.org/data/taxon/8WLMF) – simple: Roeser – full: Tricholemma subgen. Roeser
[Isogonium (Kützing) de Bary](https://www.catalogueoflife.org/data/taxon/55GX) – simple: Kuetzing – full: Isogonium subgen. Kuetzing
[Euptilota (Kützing) Kützing, 1849](https://www.catalogueoflife.org/data/taxon/4GHP) – simple: Kuetzing – full: Euptilota subgen. Kuetzing
[Setiechinopsis (Backeb.) de Haas](https://www.catalogueoflife.org/data/taxon/7NDX2) – simple: Backeb. – full: Setiechinopsis subgen. Backeb.

Chromista genera
[Cyclotella (Kützing) de Brebisson](https://www.catalogueoflife.org/data/taxon/C9G8N) – simple: Kuetzing – full: Cyclotella subgen. Kuetzing
[Tabularia (Kützing) Williams & Round](https://www.catalogueoflife.org/data/taxon/CC8M3) – simple: Kuetzing – full: Tabularia subgen. Kuetzing
[Cyrtolophosis (Schew.)](https://www.catalogueoflife.org/data/taxon/C9L2X) – simple: Schew. – full: Cyrtolophosis subgen. Schew.
[Pyrocystis (Schütt) Lemmermann, 1899](https://www.catalogueoflife.org/data/taxon/CBWGK) – simple: Schuett – full: Pyrocystis subgen. Schuett

Chromista families
[Anaulaceae (Schütt) Lemmermann](https://www.catalogueoflife.org/data/taxon/C8T39) – simple: Schuett – full: Anaulaceae subgen. Schuett
[Triceratiaceae (Schütt) Lemmermann](https://www.catalogueoflife.org/data/taxon/CCDBB) – simple: Schuett – full: Triceratiaceae subgen. Schuett
[Pyxillaceae (Schütt) Simonsen](https://www.catalogueoflife.org/data/taxon/CBX22) – simple: Schuett – full: Pyxillaceae subgen. Schuett
[Pyrocystaceae (Schütt) Lemmermann, 1899](https://www.catalogueoflife.org/data/taxon/CBWGJ) – simple: Schuett – full: Pyrocystaceae subgen. Schuett
[Aulacodiscaceae (Schütt) Lemmermann](https://www.catalogueoflife.org/data/taxon/C94XN) – simple: Schuett – full: Aulacodiscaceae subgen. Schuett
[Stictodiscaceae (Schütt) Simonsen](https://www.catalogueoflife.org/data/taxon/CC6G8) – simple: Schuett – full: Stictodiscaceae subgen. Schuett
[Lauderiaceae (Schütt) Lemmermann](https://www.catalogueoflife.org/data/taxon/CB8G5) – simple: Schuett – full: Lauderiaceae subgen. Schuett

Protozoa family
[Cyrtolophosidiidae (Schew.)](https://www.catalogueoflife.org/data/taxon/C9L2W) – simple: Schew. – full: Cyrtolophosidiidae subgen. Schew.

**Comment by dimus** [comment]
thank you @KatjaSchulz for catching this, I am not sure yet how to fix this, because in many cases `Aus (Bus)` does mean Aus subgen Bus.

Do names like this happen for bonaty names specifically?

**Comment by KatjaSchulz** [comment]
Yes, this is a tricky one. All the examples I found were taxa under the botanical code, except for the [Cyrtolophosidiidae (Schew.)](https://www.catalogueoflife.org/data/taxon/C9L2W) example which is a really weird one that has since been removed from COL. 

One approach to fix this could be a blacklist of strings that can never be interpreted as subgenus names. I think it's pretty safe to put the author strings above on that list. But after digging some more, I also found this name: [Sigmoidotropis (Piper) A.Delgado](https://www.catalogueoflife.org/data/taxon/9CT4V). I don't think there are any subgenera named Piper, but I don't know if I would be comfortable putting that name on the blacklist.

Another approach would be to add processing of rank information to gnparser. I usually have that information for most names I am trying to parse, and I use it to double-check the gnparser results. I realize that would probably be quite a bit of work to implement.

Anyway, here are a few more names I found in the COL 2024 annual archive:

Plant genera;

[Hexaphylla (Klokov) P.Caputo & Del Guacchio](https://www.catalogueoflife.org/data/taxon/9TV6S) – simple: Klokov – full: Hexaphylla subgen. Klokov
[Parogonum (Haraldson) Desjardins & J. P. Bailey](https://www.catalogueoflife.org/data/taxon/B6D46) – simple: Haraldson – full: Parogonum subgen. Haraldson
[Ericetorum (Jermy) Li Bing Zhang & X. M. Zhou](https://www.catalogueoflife.org/data/taxon/BSJ9Z) – simple: Jermy – full: Ericetorum subgen. Jermy
[Archidasyphyllum (Cabrera) P. L. Ferreira, Saavedra & Groppo](https://www.catalogueoflife.org/data/taxon/324M) – simple: Cabrera – full: Archidasyphyllum subgen. Cabrera
[Lamyropsis (Kharadze) Dittrich](https://www.catalogueoflife.org/data/taxon/639K6) – simple: Kharadze – full: Lamyropsis subgen. Kharadze
[Sigmoidotropis (Piper) A.Delgado](https://www.catalogueoflife.org/data/taxon/9CT4V) – simple: Piper – full: Sigmoidotropis subgen. Piper
[Moquiniastrum (Cabrera) G. Sancho](https://www.catalogueoflife.org/data/taxon/5V43) – simple: Cabrera – full: Moquiniastrum subgen. Cabrera

Chromista genera:

[Hormosira (Endlichter) Meneghini, 1838](https://www.catalogueoflife.org/data/taxon/7PKS6) – simple: Endlichter – full: Hormosira subgen. Endlichter
[Syracolithus (Kamptner) Deflandre in Grassé, 1952](https://www.catalogueoflife.org/data/taxon/7RHP) – simple: Kamptner – full: Syracolithus subgen. Kamptner

**Comment by dimus** [comment]
I do have a list of Botanical genera (https://github.com/gnames/gnparser/blob/master/io/dict/data/genera_auth_icn.txt), and, if they are not ambiguous, I treat text in parentheses after genus  for bi- trinomials as authorship.  I can expand this rule to uninomials as well.

**Comment by dimus** [comment]
@KatjaSchulz would implementation of #267 help for your use case? If all names are botanical, we would not have ambiguity in parsing such names

**Comment by KatjaSchulz** [comment]
Yes, I think so. Since I am usually running comprehensive data sets through gnparser, it would be a little bit more work to separate names by code, but it would be feasible. There may be lingering problems with some microorganisms, but I think those would be negligible. Thanks!

**Comment by dimus** [comment]
Ups, did not mean to close this one, reopening...

Some plant names are now recognized, some still have problems, and Chromista authors are not recognized yet.

There is a new option: `code`. It allows to force names to be parsed by ICN rules:

https://parser.globalnames.org/api/Hormosira%20(Endlichter)%20Meneghini,%201838?code=bot

https://parser.globalnames.org/?code=botanical&format=html&names=Syracolithus+%28Kamptner%29+Deflandre+in+Grass%C3%A9%2C+1952&with_details=on

**Comment by dimus** [comment]
I think we can close this one, as we do have a workaround and I do not see a better way to solve it.
